app/services/cloud_topology_service.rb

```diff
@@ -9,6 +9,8 @@ class CloudTopologyService < TopologyService
     :cloud_tenants      => [:tags, :vms => :tags],
   ]
 
+  @kinds = %i(CloudManager AvailabilityZone CloudTenant Vm Tag)
```

Comment:
![untitle](https://cloud.githubusercontent.com/assets/649130/25342383/339f8c32-290c-11e7-8086-bf290801b5a8.JPG)
because I want to use it without instantiating the service in the views :wink: as I showed you :wink: